src/components/WeekDayPicker/index.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useUniqueIdentifier } from '../../libs/hooks';
+import RequiredAsterisk from '../RequiredAsterisk';
+import RenderIf from '../RenderIf';
+import WeekDayItems from './weekDayItems';
+import { StyledFieldset, StyledLabel, StyledTextError } from './styled';
+
+const WeekDayPicker = props => {
```

Comment:
For convention let's use function declaration, only exceptional case like using React.forwardRef
Also, fix this in  helper functions